polynote-frontend/polynote/ui/component/notebook.ts

```diff
@@ -735,12 +735,12 @@ export class NotebookUI extends UIMessageTarget {
                         const params = sig.parameters.map(param => {
                             return {
                                 label: param.typeName ? `${param.name}: ${param.typeName}` : param.name,
-                                documentation: param.docString
+                                documentation: param.docString || undefined
```

Comment:
Interesting, I would have expected typescript to have caught this. Do the types need to be updated anywhere?